Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:12-alpine
+WORKDIR /app
+COPY . .
+RUN npm install
```

Comment:
For now it's ok, but for production build you'd want only necessary files to run the service (to reduce docker image size) - for example using docker "multi-stage build" and then in final stage installing only main dependencies - `npm install --production` and running (usually) ./dist folder (containing JS files transpiled from TS).

https://docs.docker.com/develop/develop-images/multistage-build/